common/build/eslint-config-fluid/printed-configs/default.json

```diff
@@ -186,7 +213,7 @@
             "error"
         ],
         "@typescript-eslint/no-parameter-properties": [
-            "off"
+            "warn"
```

Comment:
I disagree with no-parameter-properties, would keep this off.

Reply:
seems like this would cause a ton of warnings in the repo as well

Reply:
I may be in the minority, but parameter properties annoy the crap out of me when I'm trying to edit code I didn't write. "Where the heck is this property even declared?" Why do you like them?

Reply:
> seems like this would cause a ton of warnings in the repo as well

No more than the no-unsafe-\* rules. 😄

Reply:
To be clear, others like @markfields have brought this up too, so the evidence is mounting that I should change this and learn to love it. 😄